### ISSUE TITLE
Can macs2 be used as a general peakcaller?

### ISSUE THREAD

**Issue by MasterChief1O7** [macs3-project/MACS#480]
Hi,

Can MACS2 be used as a peak caller on different genomic datasets like GRO-seq or PRO-seq or PRO-cap? There I just want to know where are the peaks without any shifting. So can I use MACS2 just as a normal peakcaller? If yes can you suggest which parameters shall I use to achieve that?

Also, will the results differ if I use bam or bedgraph as input file?

Thanks

**Comment by taoliu** [comment]
@MasterChief1O7  Yes. It can be used as a general 'peak caller'. You can get some hints from this article: https://github.com/macs3-project/MACS/wiki/Advanced%3A-Call-peaks-using-MACS2-subcommands

Basically, you can use 'pileup' on BAM file in a way that you prefer (whether shift/extend or not), then use 'bdgpeakcall' on the resulting bedGraph to call peak by giving a cutoff of the height.
